easybuild/easyconfigs/d/DIRAC/DIRAC-14.1-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'DIRAC'
+version = '14.1'
+versionsuffix = '-Python-2.7.10'
+
+homepage = 'http://diracprogram.org/'
+description = """The DIRAC program computes molecular properties using relativistic quantum chemical methods."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'usempi': True}
+
+# requires registration/license to download, http://dirac.chem.sdu.dk/DIRAC14/
+sources = ['%(name)s-%(version)s-Source.tar.gz']
+
+patches = ['DIRAC-%(version)s_fix-linking-issues.patch']
+
+dependencies = [
+     ('zlib', '1.2.8'),
+     ('Boost', '1.54.0', versionsuffix),
+]
+builddependencies = [('CMake', '3.4.1')]
+
+separate_build_dir = True
+
+configopts = "-DENABLE_MPI=ON -DCMAKE_BUILD_TYPE=release"
+prebuildopts = "unset LDFLAGS && "
+preinstallopts = prebuildopts
+
+# see http://diracprogram.org/doc/release-14/installation/testing.html
+#runtest = 'test'
```

Comment:
they take forever, and require env vars to be set, see http://diracprogram.org/doc/release-14/installation/testing.html